core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -303,6 +228,21 @@ <h2 ng-if="collectionPlaythrough.hasFinishedCollection()" class="oppia-page-head
                  version="1.1"
                  style="position: absolute; left: 50%; transform: translate(-50%, 195px);"
                  ng-click="scrollToLocation('mobile-path-anchor-' + $index); updateExplorationPreview(node.getExplorationId());">
+              <image ng-if="!collectionPlaythrough.hasStartedCollection() && $index===0"
+                     x="4"
+                     y="90"
+                     height="80"
+                     width="80"
+                     ng-attr-xlink:href="<[getStaticImageUrl('/general/collection_start_here_arrow.svg')]>" 
+                     xlink:href=""></image>
+              <text ng-if="!collectionPlaythrough.hasStartedCollection() && $index===0"
+                    x="34"
+                    y="137"
+                    alignment-baseline="middle"
+                    font-family="Capriola, Roboto, Arial, sans-serif"
+                    font-size="12"
```

Comment:
Hm, this is a bit on the small side for the mobile view, don't you think? At least that's what it looks like to me when I preview it in the dev server...